.travis.yml

```diff
@@ -21,6 +21,7 @@ env:
     - CASK=puppet-agent
```

Comment:
Our list of test platforms at the top seems to be getting old. May need to update it.